src/frontend/dune

```diff
@@ -6,7 +6,7 @@
 
 (library
  (name Frontend)
- (libraries bantorra cooltt.basis cooltt.cubical cooltt.core menhirLib)
+ (libraries bantorra cooltt.basis cooltt.cubical cooltt.core ezjsonm menhirLib)
```

Comment:
@favonia How is the ch'i of this library? ok to include? I can't remember what we've used in the past.

Reply:
`ezjsonm` is okay. In general, JSON without numbers is okay.

Reply:
This is the same library that was used in `redtt`, which is why I chose it in the first place 🙂

Reply:
ok cool! ☺️